slack.json

```
@@ -0,0 +1,21 @@
+{
+  "runtime": "deno",
+  "hooks": {
+    "build": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-write --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts",
+    "get-manifest": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.5/mod.ts --manifest",
+    "get-scripts": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://raw.githubusercontent.com/slackapi/deno-slack-builder/get-scripts/src/get-scripts.ts",
+    "start": "deno run -q --unstable --allow-read --allow-net https://deno.land/x/deno_slack_runtime@0.0.3/mod.ts"
```

I'm assuming this is the `run` command, but we're going to need to include the import map
```suggestion
    "start": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_runtime@0.0.3/mod.ts"
```